src/native/linux_wayland.rs

```diff
@@ -397,7 +397,7 @@ unsafe extern "C" fn registry_add_object(
                 1,
             ) as _;
         }
-        "zxdg_decoration_manager" => {
+        "zxdg_decoration_manager_v1" => {
```

Comment:
We should probably keep both, zxdg_decoration_manager and zxdg_decoration_manager_v1?

Reply:
Hmm, both compositors I have (sway & kwin) use zxdg_decoration_manager_v1.

Reply:
Done